features/fields_extensie.feature

```diff
@@ -1,126 +1,544 @@
+      | fields              | burgerservicenummer,geslachtsaanduiding |
+      Dan bevat de response de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding |  M        |
+      En bevat de response geen andere properties
+
+    Scenario: Hele groep wordt gevraagd
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      En de persoon heeft de volgende naam gegevens
+      | naam                      |
+      | voornamen                 | Johannnes Geurt |
+      | geslachtsnaam             | Janssen         |
+      | aanduidingNaamgebruik     | E               |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                  |
+      | type                | RaadpleegMetBurgerservicenummer         |
+      | burgerservicenummer | 999994086                               |
+      | fields              | burgerservicenummer,naam                |
+      Dan bevat de response de volgende gegevens
+      | naam                       | waarde          |
+      | burgerservicenummer        | 999994086       |
+      | naam.voornamen             | Johannnes Geurt |
+      | naam.geslachtsnaam         | Janssen         |
+      | naam.aanduidingNaamgebruik | E               |
+      En bevat de response geen andere properties
+
+    Scenario: Een of enkele attributen binnen een groep worden gevraagd
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      En de persoon heeft de volgende naam gegevens
+      | naam                      |
+      | voornamen                 | Johannnes Geurt |
+      | geslachtsnaam             | Janssen         |
+      | aanduidingNaamgebruik     | E               |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                  |
+      | type                | RaadpleegMetBurgerservicenummer         |
+      | burgerservicenummer | 999994086                               |
+      | fields              | burgerservicenummer,naam.voornamen,naam.aanduidingNaamgebruik |
+      Dan bevat de response de volgende gegevens
+      | naam                       | waarde          |
+      | burgerservicenummer        | 999994086       |
+      | naam.voornamen             | Johannnes Geurt |
+      | naam.aanduidingNaamgebruik | E               |
+      En bevat de response geen andere properties
+
+## Het inregelen van de autorisatie is toch iets dat de gemeente zelf doet ?
+## Dit hoor dat toch niet thuis in de fields feature ?  Vooralsnog overgenomen uit de oude fields-feature, maar moet volgens mij weg.
```

Comment:
ik denk dat we dit wel moeten beschrijven/standaardiseren. Niet alleen een gebruiker binnen de gemeente, maar ook een aansluiter (gemeente) heeft beperkte autorisaties. Bijvoorbeeld wanneer een gemeente een buitengemeentelijk persoon opvraagt met fields=nationaliteiten.redenOpname of fields=partners.geslachtsaanduiding. En er sluiten ook niet-gemeenten aan, die ook beperkte autorisaties hebben.

Reply:
we moeten m.i. een duidelijke uitspraak doen (in dit feature of een apart feature autorisatie) over wat moet gebeuren wanneer gevraagd wordt om een gegeven waarvoor men niet geautoriseerd is.
1. foutmelding niet geautoriseerd
2. antwoord leveren zonder niet-geautoriseerde gevraagde gegeven